### PR TITLE
Add example workflow to README

### PR DESCRIPTION
**Pull request by vchernin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,55 @@ options for you and run the image using `podman`:
...
+name: Check for updates
+on:
+  schedule: # for scheduling to work this file must be in the default branch
+  - cron: "0 * * * *" # run every hour
+  workflow_dispatch: # can be manually dispatched under GitHub's "Actions" tab 
+
+jobs:
+  flatpak-external-data-checker:
+    runs-on: ubuntu-latest
+
```
Comment (vchernin) [inline README.md:96]: originally this had:
```
container:
      image: ghcr.io/flathub/flatpak-external-data-checker
```
after `runs-on`.

But as far as I can tell, it isn't needed.

File: README.md [hunk 2]
```diff
@@ -69,6 +69,55 @@ options for you and run the image using `podman`:
...
+      matrix:
+        branch: [ master ] # list all branches to check
+    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ matrix.branch }}
+
+      - uses: docker://ghcr.io/flathub/flatpak-external-data-checker:latest
+        env:
```
Comment (vchernin) [inline README.md:107]: does this actually need so many env variables for git?

File: README.md [hunk 3]
```diff
@@ -69,6 +69,55 @@ options for you and run the image using `podman`:
...
+
+To stop Flathub's tool from checking your repo, add `"disable-external-data-checker": true` to `flathub.json` in the default branch.
+
+### Custom workflow
+
+Alternatively, you can use own workflow. This can be useful if e.g. wanting to update non-default branches.
+
+Put this yaml file under `.github/workflows`, e.g. put it in `.github/workflows/update.yaml`. Ensure to put the correct path to the manifest in the last line.
+
+<details><summary>GitHub update workflow</summary><br/>
```
Comment (gasinvein) [inline README.md]: I don't think we need to collapse the snippets. The readme is already long enough and requires ToC to navigate, so the snippets won't change much.

File: README.md (hunk 1 above)
Comment (gasinvein) [reply README.md:96]: Indeed, it's not needed unless you want to run `flatpak-external-data-checker` command directly inside the container.

File: README.md (hunk 3 above)
Comment (vchernin) [reply README.md]: Yeah, I agree it doesn't help much with readme length. Removed the collapsing.

File: README.md [hunk 4]
```diff
@@ -69,6 +69,52 @@ options for you and run the image using `podman`:
...
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ matrix.branch }}
+
+      - uses: docker://ghcr.io/flathub/flatpak-external-data-checker:latest
+        env:
+          GIT_AUTHOR_NAME: Flatpak External Data Checker
+          GIT_COMMITTER_NAME: Flatpak External Data Checker
+          GIT_AUTHOR_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
```
Comment (wjt) [inline README.md:111]: Where does this magic number come from?

Comment (vchernin) [reply README.md:111]: That email is just github actions' user, the number is the user id. So the [commit author is github actions](https://github.com/vchernin/com.github.wwmm.easyeffects/pull/17/commits/f628ac0e8105f5b101830bf39c5754a786ce6731), not [the user who commited the action file](https://github.com/flathub/com.github.wwmm.easyeffects/pull/127/commits/36d38913628796f85c2e14905ae66c33cda1a1a5).

I found it "documented" [here](https://github.community/t/github-actions-bot-email-address/17204/6). I guess a dummy "flatpakbot" account  might look fancier.

Comment (wjt) [reply README.md:111]: Indeed it might.

Could you add a comment explaining this?

Comment (vchernin) [reply README.md:111]: > Could you add a comment explaining this?

done

**Review by wjt — APPROVED** [review]

**Pull request merged**